src/main/java/gameframework/game/GameConfiguration.java

```diff
@@ -97,8 +97,11 @@ public OverlapProcessor createOverlapProcessor() {
 	}
 
 	public GameUniverse createUniverse() {
-		return new GameUniverseDefaultImpl();
+		return new GameUniverseDefaultImpl(new GameData(this));
```

Comment:
you could call the other `createUniverse()` method instead. This makes it explicit that `new GameData(this)` fills a default parameter.